physical/swift.go

```diff
@@ -63,12 +63,23 @@ func newSwiftBackend(conf map[string]string, logger log.Logger) (Backend, error)
 		tenant = conf["tenant"]
 	}
 
+	domain := os.Getenv("OS_USER_DOMAIN_NAME")
+	if domain == "" {
+		domain = conf["domain"]
+	}
+	tenantDomain := os.Getenv("OS_PROJECT_DOMAIN_NAME")
```

Comment:
Given that this is new stuff, it feels like the conf `"tenant-domain"` should use `"project-domain"` if the old nomenclature was deprecated several releases ago. Sound reasonable?